### PR TITLE
2643 - va-file-input-multiple

### PR DESCRIPTION
**Pull request by nickjg231**
## Chromatic

https://2643-file-input-multi--65a6e2ed2314f7b8f98609d8.chromatic.com

## Description
Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2643
This PR creates the `va-file-input-multiple` component. It allows users to upload multiple files into a single input. There are additional features available such as error handling per-file and including additional information in each file input.

## QA Checklist
- [x] Component maintains 1:1 parity with design mocks
- [x] Text is consistent with what's been provided in the mocks
- [x] Component behaves as expected across breakpoints
- [ ] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [ ] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [ ] Tab order and focus state work as expected
- [ ] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [ ] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [ ] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Screenshots
image
image
image

## Acceptance criteria
- [ ] QA checklist has been completed
- [ ] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/web-components/src/components/va-file-input/test/va-file-input.e2e.ts [hunk 1]
```diff
@@ -60,25 +60,6 @@ describe('va-file-input', () => {
     expect(requiredSpan).not.toBeNull();
   });
 
-  // Not supporting multiple for now
```
Comment (ataker) [inline packages/web-components/src/components/va-file-input/test/va-file-input.e2e.ts:63]: Remove `multiple` tests from single file input.

File: packages/web-components/src/components/va-file-input/test/va-file-input.e2e.ts [hunk 2]
```diff
@@ -207,28 +169,22 @@ describe('va-file-input', () => {
...
     const page = await newE2EPage();
     await page.setContent(`<va-file-input buttonText="Upload a file" uswds />`);
 
     const fileInput = await page.find('va-file-input >>> input');
     expect(fileInput.getAttribute('accept')).toBeFalsy();
   });
 
-  // Skipping due to test flakiness, but this event does work in the browser
-  it.skip('v3 emits the vaChange event only once', async () => {
+  it('v3 emits the vaChange event only once', async () => {
```
Comment (ataker) [inline packages/web-components/src/components/va-file-input/test/va-file-input.e2e.ts:180]: Was able to get this to no longer be flaky (from my local tests)

File: packages/web-components/src/components/va-file-input/va-file-input.tsx [hunk 3]
```diff
@@ -89,6 +89,12 @@ export class VaFileInput {
    */
   @Prop() headerSize?: number;
 
+  /**
+   * DST only prop
+   * removes extraneous display for multiple file input
+   */
+  @Prop() headless?: boolean = false;
+
```
Comment (ataker) [inline packages/web-components/src/components/va-file-input/va-file-input.tsx:97]: Needed to add a prop that tells it not to render everything so multiple file input looks correct

File: packages/web-components/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/web-components",
-  "version": "9.2.0",
+  "version": "9.3.0",
```
Comment (micahchiang) [inline packages/web-components/package.json:3]: This can go to 10.0.0 since we're adding a new component

**Review by micahchiang — APPROVED** [review]
Looks alright to me! I'd love to hear more about how you all solved this at next week's mob session.

**Pull request merged**